packages/dapp/src/const/cookie3.ts

```diff
@@ -0,0 +1,6 @@
+import { UserOptions } from '@cookie3/analytics';
+
+export const cookie3Config: UserOptions = {
+  siteId: 405,
+  staging: import.meta.env.MODE !== 'production',
```

Comment:
Using it in localhost, it´s changing the url accordingly:

![image](https://github.com/lifinance/jumper.exchange/assets/43956540/bdffc761-7371-482f-b5cf-32d048a8a8f7)